### DISCUSSION TITLE
PAC Import with Solution file falling

### DISCUSSION THREAD

**Discussion by surajraja** [microsoft/powerplatform-build-tools#843] [Q&A]
We have a Solution that we export from Dev with all the connectors included in it, When we import the Solution to a Prod environment with the updated setting file getting an error.

PS C:\powerautomate> pac solution import --path .\TestFavColor_managed.zip --environment e9789ab-2052-e20a-a396-23370e257811 --settings-file .\TestFavColor.json
Connected as raja.shanmugavelan@xxx.com
Connected to... Forms Prod

Solution Importing...
Microsoft PowerPlatform CLI
Version: 1.31.6+g9147a23
Online documentation: https://aka.ms/PowerPlatformCLI
Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions

Error: An unexpected error occurred.
    An unexpected error occurred.

Setting Json file

{
  "EnvironmentVariables": [],
  "ConnectionReferences": [
    {
      "LogicalName": "new_sharedmicrosoftforms_14c60",
      "ConnectionId": "47b02571eb6f4c2f9ef51dd143cfa75c",
      "ConnectorId": "/providers/Microsoft.PowerApps/apis/shared_microsoftforms"
    },
    {
      "LogicalName": "new_sharedoffice365_d4a6c",
      "ConnectionId": "b32530f5f92843d7868e0d116132ac84",
      "ConnectorId": "/providers/Microsoft.PowerApps/apis/shared_office365"
    }
  ]
}

**Comment by devkeydet** [comment]
Can you try to import the same solution manually and let us know if you get the same error?  If you do, then it's not an issue with pac and you should contact support who can help route this to the Dataverse team.

**Reply by surajraja** [reply]
Hi,

Yes, we tried that, and we got the same error, without specifying the "ConnectionId" as an empty string "" The solution is imported successfully, How do we get the correct "ConnectionId" for our Production environment where we are exporting the solution to.

**Reply by johnhipolito** [reply]
Were you ever able to get this resolved? I am running into it as well. The same exact scenario.

**Reply by CodingSinceThe80s** [reply]
I can second that, getting the same error with 1.39. Is there any way to get clear error statements beyond "an unexpected errror has occurred"?

**Reply by johnhipolito** [reply]
For my specific issue, I figured out that the guid/s I use for the connection reference id in the deploymentSettings.json file, needs to be the guid of a connection (in my case a SharePoint connection) at the destination where you are importing. This resolved it for me.
